### PR TITLE
Update the website (2021)

### PR DESCRIPTION
**Pull request by weikengchen**
This is a draft PR. Feedback and info would be useful.

What has been done:
- Add Natacha Crooks into a faculty on security
- Say goodbye to Prof. Tygar.
- Update the list of postdocs
- Update the list of students
- Update certain website links

Yet, a detailed pass might be needed, and I have strong feelings that we are still missing some people (e.g., master students?). And a number of links are in `people.eecs.berkeley.edu` which would expire soon. Also, Rishabh is graduating soon.  

Also, new courses need to be added.

### CODE REVIEW

**Comment by weikengchen** [comment]
Okay. All the pending updates have been done.

One quick question: do we include 2-year masters? do we include 5th-year masters? any people we are missing?

File: index.html [hunk 1]
```diff
@@ -101,14 +88,18 @@ <h4>Percipient postdocs
 							<br>&amp; stellar staff</h4>
 						<ul>
 							<li>
-								<a href="https://www.linkedin.com/in/alisafrik">Alisa Frik</a>
```
Comment (nmalkin) [inline index.html:104]: Alisa's still here. She does have a new website, though: https://www1.icsi.berkeley.edu/~afrik/

File: index.html [hunk 2]
```diff
@@ -119,7 +110,7 @@ <h4>Gracious grad students</h4>
 						<!-- Please keep entries alphabetical by last name. -->
 						<ul>
 							<li>
-								Noura Alomar
```
Comment (nmalkin) [inline index.html:122]: Let's use this page instead for Noura: https://blues.cs.berkeley.edu/people/noura_alomar/

File: index.html [hunk 3]
```diff
@@ -298,30 +278,41 @@ <h4 title='In random order'>Amazing alumni</h4>
...
+								<a href="https://www.linkedin.com/in/benjamin-schwarz-6380b2b2/">Ben Schwarz</a>
+							</li>
 							<li>
 								<a href="https://umeshshankar.com/">Umesh Shankar</a>
 							</li>
+							<li>
+                                                                <a href="https://rshin.github.io/">Richard Shin</a>
+                                                        </li>
+							<li>
+                                                                <a href="https://people.eecs.berkeley.edu/~json/">Jeongseok Son</a>
```
Comment (nmalkin) [inline index.html]: This works, but we might want to update to http://jeongseok.net/

File: index.html [hunk 4]
```diff
@@ -332,28 +323,34 @@ <h4 title='In random order'>Amazing alumni</h4>
...
-								Alma Whitten
+								<a href="https://www.linkedin.com/in/alma-whitten-28997711/">Alma Whitten</a>
 							</li>
 							<li>
 								<a href="http://zesty.ca/">Ka-Ping Yee</a>
 							</li>
 
+							<li>
+                                                                <a href="https://people.eecs.berkeley.edu/~wzheng/">Wenting Zheng</a>
+                                                        </li>
```
Comment (nmalkin) [inline index.html]: This is a **tiny** nit, but here and in a few other places the `</li>` is misaligned with the initial `<li>`. I just have my editor autoformat the file, so if yours does this too, you might want to do that when you're making the other edits. If not, no worries, happy to fix it myself! :-)

File: index.html [hunk 5]
```diff
@@ -462,7 +459,9 @@ <h4>Undergraduate</h4>
 					<li>CS 161:
 						<strong>Computer Security</strong>
 						-
-						<a class="current" href="https://www.cs161.org/">s20</a>,
+						<a class="current" href="https://inst.eecs.berkeley.edu/~cs161/sp21/">s21</a>,
+						<a href="https://inst.eecs.berkeley.edu/~cs161/fa20/">f20</a>,
+						<a href="https://www.cs161.org/">s20</a>,
```
Comment (nmalkin) [inline index.html]: This didn't resolve for me, but `https://cs161.org/` (without the `www`) works correctly.

**Review by nmalkin — COMMENTED** [review]
Thanks a bunch for taking the time to update everything! I noticed a couple things that are worth fixing, but otherwise it looks great!

**Comment by nmalkin** [comment]
> One quick question: do we include 2-year masters? do we include 5th-year masters?

I think we're not listing masters students (sorry, folks!), because of historical reasons and to keep the list manageable.

> any people we are missing?

Several of David's students:

- Chawin https://chawins.github.io/
- An https://www.linkedin.com/in/an-ju-phd
- Norman https://normanmu.com/

**Comment by weikengchen** [comment]
Added Chawin, An, and Norman. 

I also did a quick search on the CS student directory (https://www2.eecs.berkeley.edu/Directories/Deptonly/directory-cs-grads.html?_ga=2.120283656.1445323904.1614040034-519909022.1551831751). And I think it should be complete now.

**Comment by weikengchen** [comment]
And the previously suggested updates have been applied. Great, great catch!

**Review by nmalkin — APPROVED** [review]
Awesome, thanks a bunch for doing this!

**Comment by nmalkin** [comment]
https://security.cs.berkeley.edu/ is now up-to-date!

**Pull request merged**